.claude/how-to-use.md

```
@@ -18,3 +18,11 @@ Ask Claude to use a subagent:
 "Use the code-simplifier agent to clean up the code I just wrote"
 "Use code-architect to review this design"
 ```
+
+### Using Skills
+
+For changelog cleanup and synchronization, use:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)